src/test/java/com/commercetools/sync/products/ProductSyncMockUtils.java

```diff
@@ -119,6 +121,27 @@ public static ProductDraftBuilder createProductDraftBuilder(@Nonnull final Strin
             .publish(productFromJson.getMasterData().isPublished());
     }
 
+    /**
+     * Given a {@link Set} of {@link Category} {@link ResourceIdentifier}, this method returns an instance of
+     * {@link CategoryOrderHints} containing a {@link Map}, in which each entry has category id from the supplied
+     * {@link Set} as a key and a random categoryOrderHint which is a {@link String} containing a random double value
+     * between 0 and 1 (exclusive).
+     *
+     * <p>Note: The random double value is generated by the {@link ThreadLocalRandom#current()} nextDouble method.
+     *
+     * @param categoryResourceIdentifiers set of resource identifiers of categories to build categoryOrderHints for.
+     * @return an instance of {@link CategoryOrderHints} containing a categoryOrderHint for each category in the
+     *         supplied set of category resource identifiers.
+     */
+    public static CategoryOrderHints createRandomCategoryOrderHints(
```

Comment:
I could not see any usage of this method from idea.

Reply:
In this IT https://github.com/commercetools/commercetools-sync-java/pull/398/files#diff-4b6cdd198bf924968ff292406ff87580R138